modules/jaeger-deploy-default.adoc

```diff
@@ -0,0 +1,106 @@
+
+[NOTE]
+====
+In-memory storage is not persistent, which means that if the Jaeger pod shuts down, restarts, or is replaced, that your trace data will be lost.  For persistent storage, you must use either the `production` (Elasticsearch) or `streaming` (Elasticsearch + Kafka) deployment strategy.
+====
+
+
+.Prerequisites
+
+* The Jaeger Operator must be installed.
+* Review the instructions for how to customize the Jaeger installation.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Create a new project, for example `jaeger-system`.
+
+.. Navigate to *Home* -> *Projects*.
+
+.. Click *Create Project*.
+
+.. Enter `jaeger-system` in the *Name* field.
+
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select `jaeger-system` from the Project menu.  You may have to wait a few moments for the Operators to be copied to the new project.
+
+. Click the Jaeger Operator. On the *Overview* tab, under *Provided APIs*, the Operator provides a single link.
+
+. Under *Jaeger* click *Create Instance*.
+
+. On the *Create Jaeger* page, to install using the defaults, click *Create* to create the Jaeger instance.
+
+. On the *Jaegers* page, click the name of the Jaeger instance, for example, `jaeger-all-in-one-inmemory`.
+
+. On the *Jaeger Details* page, click the *Resources* tab.  Wait until all the pods have a status of "Running" before continuing.
+
+
+[id="jaeger-create-cli_{context}"]
+== Deploying default Jaeger from the CLI
+
+Follow this procedure to create an instance of Jaeger from the command line.
+
+.Prerequisites
+
+* An installed, verified {ProductName} Operator.
+* Access to the {product-title} Command-line Interface (CLI) commonly known as `oc` that matches your {product-title} version.
+* An account with the `cluster-admin` role.
+
+Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
++
+----
+$ oc login https://{HOSTNAME}:8443
+----
+
+. Create a new project named `jaeger-system`.
++
+----
+$ oc new-project jaeger-system
+----
+
+. Create a custom resource file named `jaeger.yaml` that contains the following text:
++
+.Example jaeger-all-in-one.yaml
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger-all-in-one-inmemory
+----
++
+
+. Run the following command to deploy Jaeger:
++
+----
+$ oc create -n jaeger-system -f jaeger.yaml
+----
++
+. Run the following command to watch the progress of the Pods during the installation process:
++
+----
+$ oc get pods -n jaeger-system -w
+----
++
+You should see output similar to the following:
++
```

Comment:
You should eventually see output similar to the following?  They may see the pod in Pending and ContainerCreating state first.